tox.ini

```diff
@@ -9,6 +9,7 @@ isolated_build = true
 
 [testenv]
 changedir = .tmp/{envname}
+passenv = HOME,WINDIR,LC_ALL,LC_CTYPE,CI
```

Comment:
I think we need this. CI isn't being registered in tox run. Nothing failed but also the tests that we want to skip were not getting skipped.

Reply:
While passing CI, might as well pass a few other common ones.

Reply:
But we use none of those 🤷‍♀️

Reply:
HOME and WINDIR are common. LC doesn't hurt. If you don't like it, I can open follow-up PR to remove them. Lemme know.

Reply:
No need for a pr. But I generally like to keep the config as clean as the code, no unused stuff to linger to make future us wonder about when their usage got removed and why, etc.